src/phylo/phyplot.jl

```diff
@@ -0,0 +1,285 @@
+@enum nodesym circle=1 square=2 rect=3
+@enum textpos auto=1 alignright=2 alignleft=3 above=4
+@enum plottype phylogram=1 fan=2
+
+type NodeFormat#{S<:AbstractString}
```

Comment:
I saw that `isequal()` now uses `is()` :).

By an active constructor do you mean one with the parameters as keywords with default arguments? I do that with quite a lot of types for their inner constructor.